.github/workflows/check_upcoming_browser_versions.yml

```diff
@@ -0,0 +1,44 @@
+# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+# You need to add BrowserStack access keys in order for the tests to work. Get a BrowserStack automate account, then go
+# to the repository settings, the Secrets page and add secrets: BROWSERSTACK_USERNAME, BROWSERSTACK_ACCESS_KEY.
+# In order to receive notifications in Slack, configure a bot: (go to slack > /apps/A050S8ALJUS-platform-upcoming-browser-versions?tab=settings&next_id=0)
```

Comment:
This is an OSS project. This link won't work outside the company.

Either use a public link, or remove it. An instruction to create a Slack bot suffices.

Reply:
I didn't mean to remove the whole Slack note. Please revert and remove just the part inside the parenthesss.